src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -46,13 +46,27 @@ export class InputTimePicker implements LabelableComponent, FormComponent, Inter
   //
   //--------------------------------------------------------------------------
 
-  /** The active state of the time input */
+  /**
+   * The active state of the time input
+   *
+   * @deprecated Use open instead.
+   */
   @Prop({ reflect: true, mutable: true }) active = false;
 
   @Watch("active")
-  activeHandler(): void {
+  activeHandler(value: boolean): void {
+    this.open = value;
+  }
+
+  /** When tru, opens the `time-picker` component */
```

Comment:
Typo: `tru` => `true`. 

@benelan @geospatialem Doc-wise, would you recommend having the description be generic? `open` is used across different components.

Reply:
Good catch for doc consistency, @jcfranco! 

@anveshmekala, Would the statement below be in line with the `input-time-picker` component functionality? 

> When true, displays and positions the component. 

We've been leveraging the above line for other `open` props across components, as long as the context makes sense.

Reply:
I agree it's a bit confusing. how about  _When true, opens the `time-picker` component_  ?

Reply:
Ah, my mistake (lol, about to sign off here) - thanks for clarifying, Anvesh. 

For consistency, can we add `calcite` to the component name and a period at the end?

> When true, opens the `calcite-time-picker` component.

Reply:
sure.